tests/Unit/Comment/CommentContainerTest.php

```diff
@@ -0,0 +1,239 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Sabberworm\CSS\Tests\Unit\Comment;
+
+use PHPUnit\Framework\Constraint\LogicalAnd;
+use PHPUnit\Framework\Constraint\TraversableContains;
+use PHPUnit\Framework\TestCase;
+use Sabberworm\CSS\Comment\Comment;
+use Sabberworm\CSS\Comment\Commentable;
+use Sabberworm\CSS\Tests\Unit\Comment\Fixtures\ConcreteCommentContainer;
+use TRegx\DataProvider\DataProviders;
+
+/**
+ * @covers \Sabberworm\CSS\Comment\CommentContainer
+ */
+final class CommentContainerTest extends TestCase
+{
+    /**
+     * @var Commentable
```

Comment:
```suggestion
     * @var ConcreteCommentContainer
```

Reply:
I didn't want to make the type of the test subject any more specific than it needed to be.  But if you think it should be, I'm happy with that.